ipn/e2e_test.go

```diff
@@ -191,12 +192,16 @@ type testNode struct {
 // Create a new IPN node.
 func newNode(t *testing.T, prefix string, https *httptest.Server, weirdPrefs bool) testNode {
 	t.Helper()
-	logfe := func(fmt string, args ...interface{}) {
+
+	ulogfe := func(fmt string, args ...interface{}) {
 		t.Logf(prefix+".e: "+fmt, args...)
 	}
-	logf := func(fmt string, args ...interface{}) {
+	logfe := logger.RateLimitedFn(ulogfe, 1, 1)
```

Comment:
There's probably not much need to rate limit the logs in tests. On the other hand, maybe we should have the test fail completely if the rate limiter ever kicks in during a test, which would seem to suggest a problem.

Reply:
Fair enough, I mostly wanted to make sure that nothing broke moving further down the code when I substituted in the rate limited function.